SwiftCompilerSources/CMakeLists.txt

```diff
@@ -74,7 +74,7 @@ function(add_swift_compiler_modules_library name)
 
   set(swift_compile_options
       "-Xfrontend" "-validate-tbd-against-ir=none"
-      "-Xfrontend" "-enable-experimental-cxx-interop"
```

Comment:
Can't we do this conditionally based on the version of the compiler?

Reply:
Why would we want to conditionally select between two options which do the same thing?